docs/src/ExpandableModule.doc.js

```diff
@@ -0,0 +1,108 @@
+// @flow strict
+import React from 'react';
+import { ExpandableModule } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import Example from './components/Example.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards = [];
+const card = c => cards.push(c);
+
+card(
+  <PageHeader
+    name="ExpandableModule"
+    description="
+The ExpandableModule component renders detailed information in the expanded state and summarized information in the collapsed state.
+"
+  />
+);
+
+card(
+  <PropTable
+    Component={ExpandableModule}
+    props={[
+      {
+        name: 'items',
+        type:
+          'Array<{| title: string, icon: ?string, summary: ?Array<string>, type: ?string, iconAccessibilityLabel: ?string, children: ?React.Node |}>',
```

Comment:
Instead, could we do something like `<ExpandableModule.Item title="" icon="" />`? It would make the component more composable.